dev-haskell/pointfree/pointfree-1.1.1.9.ebuild

```
@@ -0,0 +1,33 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.8.0.0
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Tool for refactoring expressions into pointfree form"
+HOMEPAGE="https://hackage.haskell.org/package/pointfree"
+
+LICENSE=""	# FIXME: license unknown to cabal. Please pick it manually.
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=dev-haskell/haskell-src-exts-1.20:=[profile?] <dev-haskell/haskell-src-exts-1.24:=[profile?]
+	>=dev-lang/ghc-8.4.3:=
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-2.2.0.1
+	test? ( >=dev-haskell/hunit-1.6 <dev-haskell/hunit-1.7
+		>=dev-haskell/quickcheck-2.11 <dev-haskell/quickcheck-2.15 )
+"
+
+src_prepare() {
+	default
+
+	cabal_chdeps \
+		'base >= 4.5 && < 4.16' 'base >= 4.5'
+}
```

Just a reminder to please use `CABAL_CHDEPS` instead of overriding `src_prepare()`

```suggestion
CABAL_CHDEPS=(
	'base >= 4.5 && < 4.16' 'base >= 4.5'
)
```